automations/credits.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "Automation Credits"
+description: "Understand how Automation Credits are counted."
+---
+
+The Koala Automations product is available on all plans.
+The Business and Pro plans include unlimited action credits, while the other plans include
+limited credits.
+
+
+## Understanding Credits
+Each succesfull Automation run deducts 1 credit from your available credit pool.
+
+There are separate pools for Actions, Auto Outbound and Slack Alerts, so you can try
+all three products individually on your own time.
+
+Credits are deducted when any succesfull Automation is run. 
+
+## Credit Limits
+
+- Slack credits are counted per month, and reset on the first day of your subscription start date.
+- Action and Auto Outbound credits are counted per workspace, and do not reset.
+- There's no limit on the number of Automations you can create, only on the number of times they can run.
+
+The following table shows the credit limits for each plan:
+
+| Plan | Action Credits | Auto Outbound Credits | Slack Alert Credits |
+--- | --- | --- | ---
+| Free | 100 | 10 | 250 alerts /mo |
+| Starter | 100 | 10 | Unlimited |
+| Team | 100 | 10 | Unlimited |
+| Pro | Unlimited | Unlimited | Unlimited |
+| Business | Unlimited | Unlimited | Unlimited |
+
+## What happens when I run out of credits?
+
+When you run out of credits, your Automations will not run until you have more credits available.
```

Comment:
Do we wanna talk about the specifics of aob? Like autopilot is paused, but eligibility is still xhecked

Reply:
Good idea!